File: java/src/test/java/com/hachicore/demo/jooq/JooqApplicationTests.java
package com.hachicore.demo.jooq;

import com.hachicore.demo.jooq.tables.JActor;
import org.jooq.DSLContext;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
class JooqApplicationTests {

	@Autowired
	private DSLContext dslContext;

	@Test
	void test() {
		dslContext.selectFrom(JActor.ACTOR)
				.limit(10)
				.fetch();
	}

}

File: kotlin/src/test/kotlin/com/hachicore/demo/jooq/JooqApplicationTests.kt
package com.hachicore.demo.jooq

import com.hachicore.demo.jooq.tables.JActor
import org.jooq.DSLContext
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class JooqApplicationTests(
	val dslContext: DSLContext
) {

	@Test
	fun test() {
		dslContext.selectFrom(JActor.ACTOR)
			.limit(10)
			.fetch()
	}

}
